winsup/cygwin/include/cygwin/exit_process.h

```diff
@@ -262,7 +265,9 @@ exit_process_tree (HANDLE main_process, int exit_code)
                   | PROCESS_VM_OPERATION | PROCESS_VM_WRITE | PROCESS_VM_READ,
               FALSE, pids[i]);
           if (!process)
-            process = OpenProcess (PROCESS_TERMINATE, FALSE, pids[i]);
+            process = OpenProcess (
+                PROCESS_QUERY_LIMITED_INFORMATION | PROCESS_TERMINATE,
```

Comment:
Note that `PROCESS_QUERY_LIMITED_INFORMATION` is Vista+.  I assume XP support is no longer required?

Reply:
git for windows is Vista+ (https://gitforwindows.org/requirements.html) and we are 7+, so should be fine.

Reply:
I saw code calling IsWow64Process directly, but I think that was 5.2 and up, so it could have been the minimum was XP64/Server 2003.  But Vista+ seemed safe enough